MVCDemos/Views/Form/GroupedFields.cshtml

```diff
@@ -25,12 +27,14 @@
 
             items.AddGroup()
                 .Caption("Personal Data")
+                .CaptionTemplate(new JS("function(data) { return groupCaptionTemplate('accountbox',data); }"))
```

Comment:
Please check the actual demo and the icons being used. It should use 'card' instead of 'accountbox'.

http://az-jsserver.corp.devexpress.com:81/Demos-24.1/WidgetsGallery/Demo/Popup/Overview/jQuery/Light/